packages/common-all/src/utils/index.ts

```diff
@@ -6,7 +6,22 @@ import minimatch from "minimatch";
 import path from "path";
 import querystring from "querystring";
 import semver from "semver";
+import YAML from "js-yaml";
```

Comment:
yea I did not see that. I moved the new yaml utils into common-server here [`278d893` (#3649)](https://github.com/dendronhq/dendron/pull/3649/commits/278d8935684e86da23b5a54d8abd3ea3c6ea3222)

I think they would actually fit well under common-all because `js-yaml` does actually not depend on only server code. I just saw also that there exist other usages of `js-yaml` inside `common-all`. So I am considering moving that back into `common-all` and maybe out of `common-server`

Reply:
I moved the new yaml utils into `common-all` https://github.com/dendronhq/dendron/pull/3649/commits/8cc9abdbe0e46d434262d00d64752ae802184d8d

These are all the other uses of `js-yaml` which are not inside `common-server` and therefore access it because of monorepo and not because its an actual depedency.
```
packages/engine-server/src/drivers/file/schemaParser.ts|11 col 19| import YAML from "js-yaml";
packages/unified/src/remark/extendedImage.ts|8 col 19| import YAML from "js-yaml";
packages/common-server/src/filesv2.ts|22 col 36| import YAML, { JSON_SCHEMA } from "js-yaml";
packages/common-server/src/files.ts|3 col 19| import YAML from "js-yaml";
packages/plugin-core/src/web/injection-providers/getWorkspaceConfig.ts|3 col 19| import YAML from "js-yaml";
packages/plugin-core/src/web/test/helpers/WorkspaceHelpers.ts|3 col 19| import YAML from "js-yaml";
packages/unified/src/decorations/diagnostics.ts|12 col 19| import YAML from "js-yaml";
```

I find this use https://github.com/Thinkmill/manypkg#checks which is a kind of monorepo linter that would prevent these kind of things.

Reply:
I would suggest to fix them, but in a seperate PR.

Reply:
sounds good